packages/blockchain/xdc/src/lib/xdc.util.ts

```diff
@@ -0,0 +1,8 @@
+export const xdcUtil = {
+  toHex: (address: string) => {
+    return `0x${address.substring(3)}`
```

Comment:
maybe it is better to check if it starts from prefix we want to remove. to avoid misuse)